<launch>
  <!-- Configuracion de Gazebo -->
  <env name="GAZEBO_MODEL_PATH" value="${GAZEBO_MODEL_PATH}:$(find gazebo_sfm_plugin)/media/models"/>
  <env name="GAZEBO_RESOURCE_PATH" value="${GAZEBO_RESOURCE_PATH}:$(find gazebo_sfm_plugin)/media/models"/>
  
  <!-- Mundo Gazebo -->
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find gazebo_sfm_plugin)/worlds/cafe3.world"/>
  </include>

  <node name="task_coordinator" pkg="gazebo_sfm_plugin" type="task_coordinator" output="screen">
      <param name="num_pedestrians" type="int" value="3" />  <!-- 3 humanos -->
      <param name="num_robots"     type="int" value="0" />  <!-- 1 robots -->
  </node>

</launch>